_components/pages/XRPLUseCases.tsx

```diff
@@ -0,0 +1,217 @@
+      id: "raisedinspace",
+      name: "Raised in Space",
+      title: "Music",
+      description:
+        "Raised in Space is a music/tech investment group focused on raising the value of music, innovating across the entire value chain of the music industry.",
+    },
+    {
+      link: "https://xrpl-labs.com/",
+      id: "xrpl-labs",
+      name: "XRPL Labs",
+      title: "Applications",
+      description:
+        "From cold storage to apps for signing transactions, XRPL Labs is dedicated to building software on the XRP Ledger.",
+    },
+    {
+      link: "https://xrplorer.com/",
+      id: "xrplorer",
+      name: "Xrplorer",
+      title: "Security",
+      description:
+        "Xrplorer offers services and tools that help prevent and combat fraudulent activity on the XRPL as well as custom APIs and analytics that supplement the XRPL APIs where they are not enough.",
+    },
+  ];
+  const uses: Array<UseCaseCategoryInfo> = [
+    {
+      id: "micropayments",
+      title: "Micropayments",
+      description:
+        "Developers are using the XRP Ledger to build innovative products for gaming, content, and web monetization, among other applications where currency is at the center.",
+    },
+    {
+      id: "wallets",
+      title: "Cryptocurrency Wallets",
+      description:
+        "Use the Ledger to build digital wallets to store private and public passwords and interact with various blockchains to send and receive digital assets, including XRP.",
+    },
+    {
+      id: "exchanges",
+      title: "Exchanges",
+      description:
+        "Build sophisticated exchanges where users can invest and trade crypto and non-blockchain assets such as stocks, ETFs, and commodities.",
+    },
+    {
+      id: "stablecoins",
+      title: "Stablecoins",
+      description:
+        "Financial institutions can use Issued Currencies to issue stablecoins on the XRP Ledger. XRPL’s integrated decentralized exchange (DEX) allows neutral, counterparty-free digital assets to be seamlessly exchanged to and from “issued assets,” including stablecoins.",
+    },
+    {
+      id: "nft",
+      title: "NFTs",
+      description:
+        "XRPL allows issuance of IOUs which can represent a currency of any value, which can be extended to the issuance of non-fungible tokens (NFTs).",
+    },
+    {
+      id: "defi",
+      title: "DeFi",
+      description:
+        "Provide access to financial products and services online in a decentralized and borderless manner on XRPL, with decentralized smart contract protocols replacing the traditional role of financial institutions.",
+    },
+    {
+      id: "cbdc",
+      title: "CBDCs",
+      description:
+        "The CBDC Private Ledger provides Central Banks a secure, controlled, and flexible solution to issue and manage Central Bank Issued Digital Currencies (CBDCs).",
+    },
+  ];
+
+  return (
+    <div className="overflow-hidden page-uses landing-builtin-bgr">
```

Comment:
In light mode, I'm seeing a white-text-on-white-background problem for the "use case" paragraph elements in my local dev. The exact element is on line 174 below. (I believe this can be a "sometimes problem" because if two CSS rules use the same level of specificity, whichever comes last wins, and it seems like Redocly doesn't load CSS in the same way on all environments.)

The way I went about fixing something similar on the XRPL Overview is to add a `styled-page` class to this top-level div, and add a style rule that overrides Redocly's paragraph styling in light mode. I think you can use my fix by adding `styled-page` here:

```suggestion
    <div className="overflow-hidden page-uses landing-builtin-bgr styled-page">
```